mflowgen/common/fc-netlist-fixing/outputs/netlist-fixing.tcl

```diff
@@ -42,6 +42,8 @@ attachTerm -noNewPort ANAIOPAD_ext_Vcal AIO ext_Vcal
 
 # Add port for each iphy-to-bump connection
 set bump_to_iphy [list \
+  ext_tx_outp \
+  ext_tx_outn \
```

Comment:
This is incorrect. These signals should be int he iphy_to_bump list.

Reply:
> This is incorrect. These signals should be in the iphy_to_bump list.

Thanks for fixing this (plus the other changes you made)